chrome/formlock.js

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
* Most of the functionality seems to be generic, is it really Chrome-specific?
* The file needs a license, have you been able to learn more than:

sure, Formlock is open source - https://github.com/ostarov/Formlock>

Please cite our work properly of course 

* We need to distinguish between yours code and the original

Reply:
I've made it like this for the time being, because I had some problems while running it on FIrefox
Yes, the author wrote that I could use it as long as I marked his code and mentioned his name and link to original repo in the comments